src/hooks/usePlanByEpg.ts

```diff
@@ -32,8 +32,11 @@ const usePlanByEpg = (channels: EpgChannel[], sidebarWidth: number, itemHeight:
 
   // this mechanism updates the EPG component range when leaving the page open for a longer period
   // the useEpg hook doesn't accept a formatted date and re-renders when not memoize the start and end dates
-  const date = startOfDay(new Date()).toJSON();
-  const [startDate, endDate] = useMemo(() => [startOfDay(new Date()), endOfDay(new Date())], [date]);
+  // @todo ideally we want to render the schedule X hours before and after the current time, but this doesn't work (yet)
```

Comment:
From the meeting today, it sounds like it might work or might work better at least to try the +/- approach from now, but rounded to the nearest whole hour. Karol said that fractional hours don't work well.

Reply:
Ah nice, I will try that immediately

Reply:
Here are my findings so far:

1) It does work (the "now" button and line work properly)
2) The timeline does render the wrong times, but this can be fixed in our custom component
3) Increasing the +/- hours from 12 to 24 seem to result in the same problem
4) When the `startDate` lands on the previous day it also results in the same problem. This is when the current time is 1:00 AM and the start time will become 1:00 PM the previous day.

My conclusion, for now, is to keep using the `startOfDay` and `startOfTomorrow` functions until we get this sorted out in the planby component.

Reply:
Agreed. Thanks for testing it!